src/assets/scss/global/page-layout.scss

```diff
@@ -16,6 +16,13 @@ body.scroll-locked {
 	}
 }
 
+body.scroll-locked-small-only {
+	@include breakpoint(small only) {
+		-webkit-overflow-scrolling: auto;
+		overflow: hidden;
```

Comment:
I added a class that only blocks scrolling for mobile since we don't want to block scrolling on desktop/tablet